programs/Makefile

```diff
@@ -286,7 +290,7 @@ ssl/ssl_server2$(EXEXT): ssl/ssl_server2.c $(SSL_TEST_DEPS)
 	echo "  CC    ssl/ssl_server2.c"
 	$(CC) $(LOCAL_CFLAGS) $(CFLAGS) ssl/ssl_server2.c $(SSL_TEST_OBJECTS) $(LOCAL_LDFLAGS) $(LDFLAGS) -o $@
 
-ssl/ssl_context_info$(EXEXT): ssl/ssl_context_info.c test/query_config.o $(DEP)
+ssl/ssl_context_info$(EXEXT): ssl/ssl_context_info.c test/query_config.o test/query_config.h $(DEP)
```

Comment:
minor but is this dependency really necessary ? ssl/ssl_context_info depends on test/query_config.o which depends on test/query_config.h. Isn't that enough.

Reply:
It's not necessary, but I need to do extra effort to realize that it isn't. Building `ssl_context_info$(EXEXT)` from requires both `query_config.h` (when compiling) and `query_config.o` (when linking). I don't think removing the `.h` dependency and adding a comment explaining why the `.h` dependency wasn't needed would be an improvement.

Reply:
ok, fine by me.